migrator/README.md

```diff
@@ -0,0 +1,53 @@
+Migrating Elasticsearch services with snapshots
+===============================================
+
+The following with example CF deployment is an overview of migrating to the new AWS Elasticsearch service using a snapshot from the legacy Elasticsearch service.
+
+## Getting started
+
+The following example uses three services to create, store, and restore an Elasticsearch snapshot. The following services created for this example are named `old-es-service` (legacy elasticsearch56 service), `new-es-service` (new aws-elasticsearch service), and `snapshot-s3-service` (AWS S3 storage service).  Before running through this example, make sure to create the three services with the following cli commands:
```

Comment:
This suggested clarification may be a bit easier to read and more direct with the instructions:

```suggestion
The following example uses three services to create, store, and restore an Elasticsearch snapshot. In this example the services are named `old-es-service` (legacy elasticsearch56 service), `new-es-service` (new aws-elasticsearch service), and `snapshot-s3-service` (AWS S3 storage service).

For the first step, create the three services with the following CLI commands:
```

Good call on specifying the instance details to make it clear what these are!